EIPS/eip-7928.md

```diff
@@ -41,6 +42,7 @@ We introduce three new components in the block body:
 Address = ByteVector(20)
 StorageKey = ByteVector(32)
 StorageValue = ByteVector(32)
+CodeData = ByteVector(MAX_CODE_SIZE)
```

Comment:
This works, until the max code size is bumped ([EIP-7609](https://eips.ethereum.org/EIPS/eip-7907))

Reply:
All the scenarios are far away from being worst-cases, just because contracts are so expensive.
Re, the max code size, agree, we should probably bump it already but at the latest with 7609 being shipped in fusaka.